global_helpers/panther_audit.py

```diff
@@ -8,13 +8,19 @@ def build_client(resource, service, region=None):
     """Function builds resource client that assumes Panther audit role"""
     account = resource['AccountId']
     role_arn = f'arn:aws:iam::{account}:role/PantherAuditRole-{PANTHER_MASTER_REGION}'
-    sts_connection = boto3.client('sts')
+
+    sts_connection = boto3.client('sts', endpoint_url='https://sts' + fips_suffix if fips_enabled else None)
+
     acct_b = sts_connection.assume_role(
         RoleArn=role_arn, RoleSessionName="lambda_assume_audit_role")
     access_key = acct_b['Credentials']['AccessKeyId']
     secret_key = acct_b['Credentials']['SecretAccessKey']
     session_token = acct_b['Credentials']['SessionToken']
     # create service client using the assumed role credentials, e.g. S3
+
+    fips_enabled = os.getenv('ENABLE_FIPS', '').lower() == 'true'
+    fips_suffix = '-fips.' + PANTHER_MASTER_REGION + '.amazonaws.com'
```

Comment:
Pretty sure we need to define these variables _before_ we use them on line 12.

Reply:
Oops, yep

Reply:
![Screen Shot 2021-01-26 at 7 07 42 PM](https://user-images.githubusercontent.com/74208737/105923059-c6ba4380-6009-11eb-992e-65905e16cf8d.png)
![Screen Shot 2021-01-26 at 7 08 05 PM](https://user-images.githubusercontent.com/74208737/105923079-d5a0f600-6009-11eb-858c-1fffc327cf6b.png)
@nhakmiller enjoy your warm fuzzy feeling